src/main/resources/com/google/api/codegen/java/method_sample.snip

```diff
@@ -0,0 +1,157 @@
+@extends "common.snip"
```

Comment:
It's alright, I already fixed it in a holding branch (https://github.com/bjwatson/toolkit/tree/python-snippets-garrett). I carefully ensured that all your changes have been transferred into that branch, and I'll handle any incremental merges from your PR.

I find keeping method_sample.snip separate is easier than trying to visually distinguish the snippet gen code from the remainder of main.snip, even with the overhead of concurrent refactoring.

Once your PR is in master, I'll merge my holding branch into this one and then merge in master. It will be clean.